OracleSOASuite/dockerfiles/12.2.1.3/Dockerfile

```diff
@@ -14,11 +14,12 @@
 #
 # Pull base image
 # ---------------
-FROM oracle/fmw-infrastructure:12.2.1.3
+FROM oracle/fmw-infrastructure:12.2.1.3 as builder
+
 #
 # Maintainer
 # ----------
-MAINTAINER Prashanth Nagaraja <prashanth.nagaraja@oracle.com>
+LABEL maintainer="Prashanth Nagaraja <prashanth.nagaraja@oracle.com>"
```

Comment:
Is Prashanth still the maintainer or are you? Either way, the `builder` doesn't need a maintainer label.